### DISCUSSION TITLE
Check online stock

### DISCUSSION THREAD

**Discussion by orangenagy** [Ephigenia/ikea-availability-checker#42] [Q&A] [answered]
This is great. However, it seems to only check stock in stores. Is there a way to check the online stock? Or even - that would be a great feature!

**Comment by Ephigenia** [comment, marked as answer]
There’s already a python script written by @vrslev linked in this thread: https://github.com/Ephigenia/ikea-availability-checker/discussions/36 which I think covers requesting the online-shop API. If I can find time the next weeks I’ll try this out too and will integrate this into the tool.

**Reply by Ephigenia** [reply]
Short, check: https://github.com/Ephigenia/ikea-availability-checker/discussions/36

**Comment by Ephigenia** [comment]
I’ll start to create issues for all feaure requests which then can be voted one. Feel free to vote on: https://github.com/Ephigenia/ikea-availability-checker/issues/43

**Comment by Ephigenia** [comment]
Okay, sorry for confusing about this. I think it’s a better way to use the discussions for voting on feature requests using the upvote button on the original post. So I deleted the "issue".

**Reply by orangenagy** [reply]
Sounds good. Yes - so many ways to do a similar thing on github now - discussions, issues!

**Comment by bojomojo** [comment]
Would love this as well. I already automated the process to watch the stock and pushing a notification when it changes. So would love to complete it by including the onlineshop stockbas well.

**Reply by vrslev** [reply]
Checkout [vrslev/ikea-api-client](https://github.com/vrslev/ikea-api-client). Written in Python but fairly easy to use
